docs/api-ref.md

```diff
@@ -2682,6 +2682,55 @@ The schedule methods are based upon the endpoints for schedules in the REST API
 
 Source files: server/endpoint/schedules_endpoint.py
 
+#### schedule.add_to_schedule
+
+```py
+schedules.add_to_schedule(schedule_id)
+```
+
+Adds a `DataSourceItem` or a `WorkbookItem` to a schedule.
+
+
+REST API: [Add Data Source to Schedule](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_jobs_tasks_and_schedules.htm#add_data_source_to_schedule)
+
+
+
+**Parameters**
+
+Name  |  Description
+:--- | :---
+`schedule_id` | The Tableau-internal identifier of the schedule to add to. Can be retrieved from a `ScheduleItem`, e.g., by performing `schedules.get()`
+`workbook` | (Optional) A `WorkbookItem` that should be added to the schedule. Only provide either `workbook` or `datasource`, passing multiple items will be deprecated.
+`datasource` | (Optional) A `DataSourceItem` that should be added to the schedule. Only provide either `workbook` or `datasource`, passing multiple items will be deprecated.
```

Comment:
Same: should be DatasourceItem

Reply:
@bcantoni - more a question for you.  it says here that "passing multiple items will be deprecated."  So does this mean you can do that today and this functionality will be deprecated in the future?

Reply:
As I could get I from the source code, you can currently pass a workbook and a datasource item at the same time with one method call, which leads to PendingDeprecationWarning. A FlowItem can only be passed individually.

Reply:
@kramantab it does sound that way (meaning: might be deprecated in the future). I'm happy with the wording @bossenti put in here. It mentions the possible deprecation without being to specific about it.